### ISSUE TITLE
ng-bs-daterangepicker.min.js is broken

### ISSUE THREAD

**Issue by lanocturne** [luisfarzati/ng-bs-daterangepicker#51]
Somehow if ng-bs-daterangepicker.min.js is loaded the daterangepicker's predefined range selector would not show( today, custom etc, demo can be found on original bs version). Current workaround is using ng-bs-daterangepicker.js instead of the .min one.

**Comment by pieterjandesmedt** [comment]
I second @lanocturne's findings.

**Comment by wedgberto** [comment]
this is due to the injectables into the input directive are being renamed by the minification.

change
```javascript
angular.module('ngBootstrap', []).directive('input', function ($compile, $parse) {
...
```
to 
```javascript
angular.module('ngBootstrap', []).directive('input', ['$compile', '$parse', function ($compile, $parse) {
...
```
*from [Angular JS Tutorial 5](https://docs.angularjs.org/tutorial/step_05)*
>**A Note on Minification**
>Since Angular infers the controller's dependencies from the names of arguments to the controller's constructor function, if you were to minify the JavaScript code for PhoneListCtrl controller, all of its function arguments would be minified as well, and the dependency injector would not be able to identify services correctly.
>
>We can overcome this problem by annotating the function with the names of the dependencies, provided as strings, which will not get minified. There are two ways to provide these injection annotations:
>
>Create a $inject property on the controller function which holds an array of strings. Each string in the array is the name of the service to inject for the corresponding parameter. In our example we would write:
>```javascript
>function PhoneListCtrl($scope, $http) {...}
>PhoneListCtrl.$inject = ['$scope', '$http'];
>phonecatApp.controller('PhoneListCtrl', PhoneListCtrl);
>```
>Use an inline annotation where, instead of just providing the function, you provide an array. This array contains a list of the service names, followed by the function itself.
>```javascript
>function PhoneListCtrl($scope, $http) {...}
>phonecatApp.controller('PhoneListCtrl', ['$scope', '$http', PhoneListCtrl]);
>```
>Both of these methods work with any function that can be injected by Angular, so it's up to your project's style guide to decide which one you use.
>
>When using the second method, it is common to provide the constructor function inline as an anonymous function when registering the controller:
>```javascript
>phonecatApp.controller('PhoneListCtrl', ['$scope', '$http', function($scope, $http) {...}]);
>```

**Comment by alanjames1987** [comment]
This has been fixed in version 0.0.5.
